ui/app/src/components/EmptyState.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import Typography from "@material-ui/core/Typography";
+import emptyIGM from "../assets/desert.svg";
+import { defineMessages, useIntl } from "react-intl";
+
+const useStyles = makeStyles(() => ({
+  emptyContainer: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    margin: 'auto',
+    '& .title': {
+      marginTop: '20px',
+      marginBottom: '10px'
+    },
+    '& .paragraph': {
+      marginTop: '10px',
+    },
+  }
+}));
+
+const messages = defineMessages({
```

Comment:
Let's remove the alt for the image. It should be a empty string as it is a decorative element.